tests/safety/test_ford.py

```diff
@@ -158,16 +167,26 @@ def _lkas_command_msg(self, action: int):
     }
     return self.packer.make_can_msg_panda("Lane_Assist_Data1", 0, values)
 
-  # TJA command
-  def _tja_command_msg(self, enabled: bool, path_offset: float, path_angle: float, curvature: float, curvature_rate: float):
-    values = {
-      "LatCtl_D_Rq": 1 if enabled else 0,
-      "LatCtlPathOffst_L_Actl": path_offset,     # Path offset [-5.12|5.11] meter
-      "LatCtlPath_An_Actl": path_angle,          # Path angle [-0.5|0.5235] radians
-      "LatCtlCurv_NoRate_Actl": curvature_rate,  # Curvature rate [-0.001024|0.00102375] 1/meter^2
-      "LatCtlCurv_No_Actl": curvature,           # Curvature [-0.02|0.02094] 1/meter
-    }
-    return self.packer.make_can_msg_panda("LateralMotionControl", 0, values)
+  # LCA command
+  def _lat_ctl_msg(self, enabled: bool, path_offset: float, path_angle: float, curvature: float, curvature_rate: float):
+    if self.STEER_MESSAGE == MSG_LateralMotionControl:
+      values = {
+        "LatCtl_D_Rq": 1 if enabled else 0,
+        "LatCtlPathOffst_L_Actl": path_offset,     # Path offset [-5.12|5.11] meter
+        "LatCtlPath_An_Actl": path_angle,          # Path angle [-0.5|0.5235] radians
+        "LatCtlCurv_NoRate_Actl": curvature_rate,  # Curvature rate [-0.001024|0.00102375] 1/meter^2
+        "LatCtlCurv_No_Actl": curvature,           # Curvature [-0.02|0.02094] 1/meter
+      }
+      return self.packer.make_can_msg_panda("LateralMotionControl", 0, values)
+    elif self.STEER_MESSAGE == MSG_LateralMotionControl2:
+      values = {
+        "LatCtl_D2_Rq": 1 if enabled else 0,
+        "LatCtlPathOffst_L_Actl": path_offset,     # Path offset [-5.12|5.11] meter
+        "LatCtlPath_An_Actl": path_angle,          # Path angle [-0.5|0.5235] radians
+        "LatCtlCrv_NoRate2_Actl": curvature_rate,  # Curvature rate [-0.001024|0.001023] 1/meter^2
+        "LatCtlCurv_No_Actl": curvature,           # Curvature [-0.02|0.02094] 1/meter
+      }
+      return self.packer.make_can_msg_panda("LateralMotionControl2", 0, values)
```

Comment:
Think it would be more in-convention if you just override this function in the CAN FD test